Standalone (JBang) install fails

## Description

Attempting to install camel-kit as described in documentation fails



## Steps to Reproduce

1. Run `# Install camel-kit globally
jbang app install camel-kit@luigidemasi/camel-kit`
2. install fails with the following error:
`[jbang] Resolving dependencies...
[jbang]    io.github.luigidemasi:camel-kit-core:0.3.2-SNAPSHOT
[jbang] [ERROR] Could not read artifact descriptor for io.github.luigidemasi:camel-kit-core:jar:0.3.2-SNAPSHOT
[jbang] Run with --verbose or -x for more details. The --verbose or -x must be placed before the jbang command. I.e. jbang --verbose run [...]`

## Expected Behavior

Install is successful 

## Actual Behavior

The install fails, verbose logs are as follows:

```
[jbang] [0:406] Resolving artifact(s): io.github.luigidemasi:camel-kit-core:0.3.2-SNAPSHOT
[jbang] [0:406] Repositories: central=https://repo1.maven.org/maven2/
[jbang] [0:420] Resolving dependencies...
[jbang] [0:523]    io.github.luigidemasi:camel-kit-core:0.3.2-SNAPSHOT
[jbang] [1:206] Deleting folder /Users/rsubramani/.jbang/cache/jars/CamelKit.java.7e08583833ad207ccb400622c04da243c6835b092a52aa67e570a222bd881264/classes
[jbang] [1:209] Deleting folder /Users/rsubramani/.jbang/cache/jars/CamelKit.java.7e08583833ad207ccb400622c04da243c6835b092a52aa67e570a222bd881264/generated
[jbang] [1:211] [ERROR] Could not read artifact descriptor for io.github.luigidemasi:camel-kit-core:jar:0.3.2-SNAPSHOT
dev.jbang.cli.ExitException: Could not read artifact descriptor for io.github.luigidemasi:camel-kit-core:jar:0.3.2-SNAPSHOT
        at dev.jbang.dependencies.ArtifactResolver.resolveDescriptor(ArtifactResolver.java:379)
        at dev.jbang.dependencies.ArtifactResolver.getManagedDependencies(ArtifactResolver.java:348)
        at dev.jbang.dependencies.ArtifactResolver.lambda$resolve$1(ArtifactResolver.java:209)
        at java.base/java.util.stream.ReferencePipeline$7$1FlatMap.accept(ReferencePipeline.java:288)
        at java.base/java.util.ArrayList$ArrayListSpliterator.forEachRemaining(ArrayList.java:1716)
        at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:570)
        at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:560)
        at java.base/java.util.stream.ReduceOps$ReduceOp.evaluateSequential(ReduceOps.java:921)
        at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:265)
        at java.base/java.util.stream.ReferencePipeline.collect(ReferencePipeline.java:723)
        at dev.jbang.dependencies.ArtifactResolver.resolve(ArtifactResolver.java:210)
        at dev.jbang.dependencies.DependencyUtil.resolveDependencies(DependencyUtil.java:118)
        at dev.jbang.dependencies.DependencyResolver.resolve(DependencyResolver.java:66)
        at dev.jbang.source.BuildContext.resolveClassPath(BuildContext.java:118)
        at dev.jbang.source.sources.JavaSource$JavaAppBuilder$JavaCompileBuildStep.getCompileCommandOptions(JavaSource.java:97)
        at dev.jbang.source.buildsteps.CompileBuildStep.getCompileCommand(CompileBuildStep.java:81)
        at dev.jbang.source.buildsteps.CompileBuildStep.compile(CompileBuildStep.java:58)
        at dev.jbang.source.buildsteps.CompileBuildStep.build(CompileBuildStep.java:54)
        at dev.jbang.source.buildsteps.CompileBuildStep.build(CompileBuildStep.java:39)
        at dev.jbang.source.AppBuilder.build(AppBuilder.java:97)
        at dev.jbang.source.AppBuilder.build(AppBuilder.java:22)
        at dev.jbang.cli.AppInstall.install(App.java:144)
        at dev.jbang.cli.AppInstall.doCall(App.java:97)
        at dev.jbang.cli.BaseCommand.call(BaseCommand.java:149)
        at dev.jbang.cli.BaseCommand.call(BaseCommand.java:24)
        at picocli.CommandLine.executeUserObject(CommandLine.java:2031)
        at picocli.CommandLine.access$1500(CommandLine.java:148)
        at picocli.CommandLine$RunLast.executeUserObjectOfLastSubcommandWithSameParent(CommandLine.java:2469)
        at picocli.CommandLine$RunLast.handle(CommandLine.java:2461)
        at dev.jbang.cli.JBang$3.handle(JBang.java:166)
        at dev.jbang.cli.JBang$3.handle(JBang.java:161)
        at picocli.CommandLine$AbstractParseResultHandler.execute(CommandLine.java:2277)
        at picocli.CommandLine$RunLast.execute(CommandLine.java:2425)
        at picocli.CommandLine.execute(CommandLine.java:2174)
        at dev.jbang.Main.main(Main.java:28)
Caused by: org.eclipse.aether.resolution.ArtifactDescriptorException: Failed to read artifact descriptor for io.github.luigidemasi:camel-kit-core:jar:0.3.2-SNAPSHOT
        at org.apache.maven.repository.internal.DefaultArtifactDescriptorReader.loadPom(DefaultArtifactDescriptorReader.java:254)
        at org.apache.maven.repository.internal.DefaultArtifactDescriptorReader.readArtifactDescriptor(DefaultArtifactDescriptorReader.java:183)
        at org.eclipse.aether.internal.impl.DefaultRepositorySystem.readArtifactDescriptor(DefaultRepositorySystem.java:269)
        at dev.jbang.dependencies.ArtifactResolver.resolveDescriptor(ArtifactResolver.java:377)
        ... 34 more
Caused by: org.eclipse.aether.resolution.ArtifactResolutionException: The following artifacts could not be resolved: io.github.luigidemasi:camel-kit-core:pom:0.3.2-SNAPSHOT (absent): Could not find artifact io.github.luigidemasi:camel-kit-core:pom:0.3.2-SNAPSHOT in central (https://repo1.maven.org/maven2/)
        at org.eclipse.aether.internal.impl.DefaultArtifactResolver.resolve(DefaultArtifactResolver.java:473)
        at org.eclipse.aether.internal.impl.DefaultArtifactResolver.resolveArtifacts(DefaultArtifactResolver.java:261)
        at org.eclipse.aether.internal.impl.DefaultArtifactResolver.resolveArtifact(DefaultArtifactResolver.java:243)
        at org.apache.maven.repository.internal.DefaultArtifactDescriptorReader.loadPom(DefaultArtifactDescriptorReader.java:243)
        ... 37 more
Caused by: org.eclipse.aether.transfer.ArtifactNotFoundException: Could not find artifact io.github.luigidemasi:camel-kit-core:pom:0.3.2-SNAPSHOT in central (https://repo1.maven.org/maven2/)
        at org.eclipse.aether.connector.basic.ArtifactTransportListener.transferFailed(ArtifactTransportListener.java:42)
        at org.eclipse.aether.connector.basic.BasicRepositoryConnector$TaskRunner.run(BasicRepositoryConnector.java:417)
        at org.eclipse.aether.connector.basic.BasicRepositoryConnector.get(BasicRepositoryConnector.java:260)
        at org.eclipse.aether.internal.impl.DefaultArtifactResolver.performDownloads(DefaultArtifactResolver.java:537)
        at org.eclipse.aether.internal.impl.DefaultArtifactResolver.resolve(DefaultArtifactResolver.java:449)
        ... 40 more
[jbang] [1:212] If you believe this a bug in jbang, open an issue at https://github.com/jbangdev/jbang/issues
```

--- a/.github/workflows/snapshot.yml
+++ b/.github/workflows/snapshot.yml
@@ -32,3 +32,48 @@ jobs:
         env:
           MAVEN_USERNAME: ${{ secrets.MVN_CENTRAL_USER }}
           MAVEN_PASSWORD: ${{ secrets.MVN_CENTRAL_PASSWORD }}
+
+  smoke-install:
+    name: Verify snapshot installation
+    needs: deploy
+    if: github.ref == 'refs/heads/main'
+    runs-on: ubuntu-latest
+    timeout-minutes: 15
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Install snapshot in a clean JBang container
+        run: |
+          snapshot_version=$(awk -F'[<>]' '/^[[:space:]]*<version>/{print $3; exit}' pom.xml)
+          [[ "$snapshot_version" == *-SNAPSHOT ]]
+
+          for attempt in {1..10}; do
+            if docker run --rm \
+              --entrypoint /bin/bash \
+              -e SNAPSHOT_VERSION="$snapshot_version" \
+              docker.io/jbangdev/jbang-action:0.139.3@sha256:f5d2d51955d4b6c8ed005b880e6b5f48f0322585298d7e3c146f8241e2d44b01 -lc '
+                set -euo pipefail
+                export HOME=/tmp/camel-kit-smoke
+                export JBANG_DIR="$HOME/.jbang"
+                export JBANG_REPO="$HOME/.m2/repository"
+                export JAVA_TOOL_OPTIONS="-Duser.home=$HOME -Dmaven.repo.local=$JBANG_REPO"
+                export JBANG_NO_VERSION_CHECK=1
+                export PATH="$JBANG_DIR/bin:/jbang/bin:$PATH"
+
+                mkdir -p "$HOME" "$JBANG_REPO"
+                jbang trust add https://github.com/luigidemasi/camel-kit/
+                jbang app install camel-kit@luigidemasi/camel-kit
+                test "$(camel-kit --version)" = "$SNAPSHOT_VERSION"
+              '; then
+              exit 0
+            fi
+
+            if [ "$attempt" -eq 10 ]; then
+              exit 1
+            fi
+
+            echo "Snapshot not visible yet; retrying in 30 seconds"
+            sleep 30
+          done

--- a/camel-kit-main/src/main/jbang/main/CamelKit.java
+++ b/camel-kit-main/src/main/jbang/main/CamelKit.java
@@ -3,6 +3,7 @@
 //JAVA 17+
 
 //REPOS central=https://repo1.maven.org/maven2/,apache_snap=https://repository.apache.org/snapshots
+//REPOS central_snap=https://central.sonatype.com/repository/maven-snapshots/
 //DEPS io.github.luigidemasi:camel-kit-core:${camel.kit.version:0.3.2-SNAPSHOT}
 
 package main;

